### PR TITLE
OpenNEM (Independent Publisher)

### PR DESCRIPTION
**Pull request by paulculmsee**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [x ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. (please note this is an open source project with incomplete docco)
- [x ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 

image

- [x ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.

image

- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,2456 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "OpenNEM",
+    "description": "An Open Platform for Australian Electricity Market Data",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json:5]: @paulculmsee Can you please provide a more detailed description which can be understandable by everyone and it should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

File: independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,2456 @@
...
+                    "description": "regions"
+                  },
+                  "timezone": {
+                    "type": "string",
+                    "description": "timezone"
+                  },
+                  "interval_size": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "interval_size"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json:63]: @paulculmsee All the description values are expected to be free from grammatical and spell errors. Please provide valid description values for all the properties.

Docs  - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

File: independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2456 @@
...
+                              "interval_shift": {
+                                "type": "integer",
+                                "format": "int32",
+                                "description": "interval_shift"
+                              },
+                              "intervals_per_hour": {
+                                "type": "number",
+                                "format": "float",
+                                "description": "intervals_per_hour",
+                                "title": ""
```
Comment (sriyen-msft) [inline independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json:490]: @paulculmsee Empty property values can be removed until unless you have a specific reason to keep them.

File: independent-publisher-connectors/OpenNEM/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#da3b01",
+    "capabilities": [],
+    "publisher": "Paul Culmsee (sevensigma.com.au)",
+    "stackOwmer": "OpenNEM (https://opennem.org.au/about/)"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/OpenNEM/apiProperties.json:7]: @paulculmsee Can you please remove the website URLs from the `"publisher"` and `"stackOwner"` properties. We have metadata sections in the swagger where you can provide website URLs.

File: independent-publisher-connectors/OpenNEM/settings.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (sriyen-msft) [inline independent-publisher-connectors/OpenNEM/settings.json:1]: @paulculmsee Please remove `"settings.json"` file from the PR which is not required to open-source your connector.

File: independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,2456 @@
+{
```
Comment (sriyen-msft) [inline independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json:1]: @paulculmsee Please remove the other files you have added, i.e. "Screeenshot1.jpg" and "Screeenshot2.jpg" from the PR which have no references in the readme.md file.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json (hunk 1 above)
Comment (paulculmsee) [reply independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json:5]: Okay. I made a note in the certifying step that this is an open source API and they dont publish the schema, so they will be terse. Cool with that?

File: independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json (hunk 2 above)
Comment (paulculmsee) [reply independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json:63]: Okay. I made a note in the certifying step that this is an open source API and they dont publish the schema, so they will be terse. Cool with that?

File: independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json (hunk 6 above)
Comment (paulculmsee) [reply independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json:1]: To certify the intent was to add screenshots which are referred to in the PR. Is the intent to host these elsewhere?

**Comment by natalie-pienkowska** [comment]
Thanks for adding more commits, @paulculmsee! This week is Thanksgiving week, so we won't be looking at PRs until Monday, 11/29. Hope you have a great week :)

File: independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json (hunk 6 above)
Comment (sriyen-msft) [reply independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json:1]: @paulculmsee Sorry for the delay in response, You need to add them in the PR description, not in the connector folder. Please remove the screenshots to get this PR merged.

Comment (paulculmsee) [reply independent-publisher-connectors/OpenNEM/apiDefinition.swagger.json:1]: Resolved with a previous commit

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Comment by sriyen-msft** [comment]
@paulculmsee Apologies for the delay in response.
Update - Your connector is now in production deployment and reached [Test](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#region-deployment) region, and it would be completed and available in all the production regions within 3 weeks. Please let us know if you have any questions, Thanks.

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@paulculmsee Apologies for the delay in response. 

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**